pkgs/os-specific/linux/sheep-net/default.nix

```diff
@@ -0,0 +1,35 @@
+{
+  kernel,
+  kernelModuleMakeFlags,
+  stdenv,
+  basiliskii,
+  lib,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  name = "sheep_net";
+  version = basiliskii.version;
+  src = basiliskii.src;
```

Comment:
Unless this specifically needs to be updated in lockstep with basiliskii, it would be better to decouple them, as otherwise it's very surprising that updates to that could break this.

Reply:
How would you perform the decoupling? Fetch it twice?

Reply:
it's not that updates could break things, it's that updates can fix things. E.g https://github.com/kanjitalk755/macemu/pull/280